pkgs/development/haskell-modules/patches/basement-fix-32-bit.patch

```diff
@@ -0,0 +1,216 @@
+--- a/Basement/Bits.hs
++++ b/Basement/Bits.hs
+@@ -54,8 +54,12 @@
+ import Basement.Compat.Primitive
+ 
+ #if WORD_SIZE_IN_BITS < 64
++#if __GLASGOW_HASKELL__ >= 902
++import GHC.Exts
++#else
+ import GHC.IntWord64
+ #endif
++#endif
+ 
+ -- | operation over finite bits
+ class FiniteBitsOps bits where
+--- a/Basement/From.hs
++++ b/Basement/From.hs
```

Comment:
Might be better to fetchpatch from the merge request on upstream ( https://github.com/haskell-foundation/foundation/commit/38be2c93acb6f459d24ed6c626981c35ccf44095.patch )